src/openMVG/sfm/AlembicImporter.hpp

```diff
@@ -25,17 +20,14 @@ class AlembicImporter
 {
 public:
   explicit AlembicImporter(const std::string &filename);
-  ~AlembicImporter() = default;
+  ~AlembicImporter();
 
   void populate(sfm::SfM_Data &sfmdata, sfm::ESfM_Data flags_part = sfm::ESfM_Data::ALL);
 
 private:
-  bool readPointCloud(IObject iObj, M44d mat, sfm::SfM_Data &sfmdata, sfm::ESfM_Data flags_part);
-  bool readCamera(IObject iObj, M44d mat, sfm::SfM_Data &sfmdata, sfm::ESfM_Data flags_part, const index_t sampleFrame = 0);
   
-  //@todo complete the interface, also maybe parameters need to be passed by reference?
-  void visitObject(IObject iObj, M44d mat, sfm::SfM_Data &sfmdata, sfm::ESfM_Data flags_part = sfm::ESfM_Data::ALL);
-  IObject _rootEntity;
+  struct ObjImpl;
+  std::unique_ptr<ObjImpl> _objImpl;
```

Comment:
too bad the real namespace for `IObject` is `Alembic::Abc::ALEMBIC_VERSION_NS` where `ALEMBIC_VERSION_NS` is a macro with the internal version. Any idea on how to overcome this without including the header?